docs/sphinx/spelling-wordlist.txt

```diff
@@ -0,0 +1,125 @@
+Burggraf
+CFL
+CMake
+CTest
+Ekman
+Euler
+Exascale
+Fortran
+Joukowsky
+MPI
+Moeng
+Monin
+Nalu
+Navier
+Neumann
+Nicolson
+Obukhov
+Poiseuille
+Raphson
+Shubin
+viscosities
+Zalesak
+ABL
+adaptively
+advected
+amr
+amrex
+anelastic
+anisotropic
+bicgcg
+bicgstab
+bool
+cartesian
+centerline
+cgbicg
+chk
+chkp
+cmake
+codebase
+concretization
+concretize
+concretized
+concretizing
+conda
+convecting
+cpp
+datafile
+dimensionalize
+discretization
+discretization
+discretized
+discretized
+discretizes
+doxygen
+dropdown
+dt
+enstrophy
+exawind
+executables
+fidelities
+fllc
+freestream
+frontend
+geostrophic
+Godunov
+hexahedral
+hypre
+incflo
+incompressibility
+initializer
+initializers
+iteratively
+Kosovic
+levelset
+lidar
+mesoscale
+microscale
+multigrid
+namespace
+namespaces
+nd
+netcdf
+normals
+ok
+openfast
+overset
+paraview
+perturbational
+petsc
+piecewise
+plotfile
+plotfiles
+plt
+postprocessing
+Prandtl
+pre
+priori
+proj
+reStructuredText
+regrid
+regridding
+repo
+roadmap
+rosco
+Smagorinsky
+solvability
+spack
+spanwise
+spinup
+src
+subgrid
+submodule
+submodules
+teardown
+th
```

Comment:
does `th` refer to something universal?

Reply:
`nd` and `th` are used in stuff like `2nd` and `4th`. But maybe it's better for me to track those down and just change that usage. Let me see.

Reply:
there were so few, I just turned them into actual words.